homeassistant/components/hunterdouglas_powerview/switch.py

```diff
@@ -0,0 +1,130 @@
+"""Support for hunterdouglass_powerview switches."""
+
```

Comment:
https://github.com/home-assistant/core/pull/97535/files#r1280054236

Another option we have is to start an integration branch and merge each PR to it as its reviewed and than merge the integration branch at the end

Reply:
all for that if that, makes sense to me - im still relatively new to all the git features and things seems easy to mess up anything outside commit -> push 🤣 

does that require you creating a new branch and a resubmission of this PR (which is probably easier anyway to break this down)? 
Do i then just submit a PR per commit/feature for example aimed directly at that branch ?

Reply:
I created `powerview_3_integration`

https://github.com/home-assistant/core/tree/powerview_3_integration

When you have this ready, cut out the new platforms, target it to that branch.  Once we get it though the review process, we can merge each platform in a separate PR, and than we can merge the integration branch to `dev`